homeassistant/components/xiaomi_miio/translations/ro.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Please remove the added translation files. We download translations from Lokalise, so adding them here won't have any affect.

Reply:
removed the files.